executors/kafka/README.md

```diff
@@ -72,3 +73,44 @@ testcases:
     - result.messages.__len__ ShouldEqual 1
 
 ```
+
+Example with Avro:
+
+```yaml
+name: My Kafka testsuite
+version: "2"
+testcases:
+- name: Kafka test
+  description: Test Kafka
+  steps:
+  - type: kafka
+    clientType: producer
+    withSASL: true
```

Comment:
To avoid boilerplate configuration at each step, could you handle kafka connection parameters at the testsuite variable level. 

Like in the redis executor:
https://github.com/ovh/venom/blob/master/executors/redis/redis.go#L54
https://github.com/ovh/venom/blob/master/tests/redis.yml#L5

The benefit will be be able to declare only one time:
  - addrs
  - with_tls
  - with_sasl
  - with_sasl_handshaked
  - with_avro optional 
  - user
  - password 
  - kafka_version

And if those values can be overridden at the step level, it would be nice.

Reply:
I moved the kafka variables into `tests/kafka/testVariables.yml`, please let me know if that is what you meant.